cmd/minikube/cmd/start.go

```diff
@@ -521,7 +521,9 @@ func validateDriver(ds registry.DriverState, existing *config.ClusterConfig) {
 		}
 
 		if !viper.GetBool(force) {
-			exit.WithCodeT(exit.Unavailable, "Failed to validate '{{.driver}}' driver", out.V{"driver": name})
+			docURL := fmt.Sprintf("https://minikube.sigs.k8s.io/docs/drivers/%s/", strings.ToLower(name))
```

Comment:
We already show the documentation URL above, no need to recreate it and redisplay it here.